string command
int32 value
---
int32 res
string error
time stamp 